backstop/tests/table.html

```diff
@@ -1867,1876 +1242,4740 @@ <h2 id="regular">Zebra striping</h2>
+                class="iui-icon iui-sort"
+                viewBox="0 0 16 16"
+                aria-hidden="true"
+              >
+                <path d="m7 0v12.7l-3.8-3.7-1.2 1.2 6 5.8 1.2-1.2 4.8-4.6-1.2-1.2-3.8 3.7v-12.7z" />
+              </svg>
+              <div class="iui-resizer">
+                <div class="iui-resizer-bar"></div>
+              </div>
+            </div>
+            <div class="iui-cell">
+              File count
+              <button
+                class="iui-button iui-borderless"
+                aria-label="Apply filter"
+                type="button"
+              >
+                <!-- Use filter-hollow.svg from @itwin/itwinui-icons package. -->
+                <svg
+                  class="iui-icon"
+                  aria-hidden="true"
+                  viewBox="0 0 16 16"
+                >
+                  <path
+                    d="m15 1v.5l-5.6 4.7-.4.3v.5 5.5l-2 1.5v-7-.5l-.4-.3-5.6-4.7v-.5zm1-1h-16v2l6 5v9l4-3v-6l6-5z" />
+                </svg>
+              </button>
+              <!-- Use sort-down.svg from @itwin/itwinui-icons package. -->
+              <svg
+                class="iui-icon iui-sort"
+                viewBox="0 0 16 16"
+                aria-hidden="true"
+              >
+                <path d="m7 0v12.7l-3.8-3.7-1.2 1.2 6 5.8 1.2-1.2 4.8-4.6-1.2-1.2-3.8 3.7v-12.7z" />
+              </svg>
+            </div>
+            <div class="iui-cell iui-slot">
+              <button
+                class="iui-button iui-borderless"
+                aria-label="Modify columns"
+                type="button"
+              >
+                <!-- Use column-manager.svg from @itwin/itwinui-icons package. -->
+                <svg
+                  class="iui-icon"
+                  aria-hidden="true"
+                  viewBox="0 0 16 16"
+                >
+                  <path d="m1 0h2v16h-2m6-16h2v16h-2m6-16h2v16h-2" />
+                </svg>
+              </button>
+            </div>
+          </div>
+        </div>
+
+        <div class="iui-body">
+          <div class="iui-row">
+            <div class="iui-cell iui-slot">
+              <label class="iui-checkbox">
+                <input
+                  type="checkbox"
+                  id="select-all-checkboxes"
+                />
+                <span class="iui-checkbox-checkmark">
+                  <svg
+                    viewBox="0 0 16 16"
+                    aria-hidden="true"
+                    focusable="false"
+                  >
+                    <path
+                      class="iui-check-partial"
+                      d="m1 6.5h14v3h-14z"
+                    />
+                    <path
+                      class="iui-check"
+                      d="M6,14L0,8l2-2l4,4l8-8l2,2L6,14z"
+                    />
+                  </svg>
+                </span>
+              </label>
+            </div>
+            <div class="iui-cell iui-slot">
```

Comment:
In this example a row can be expanded to show content that is not following the table structure.  The expandable arrows are placed within their own column.  AFAIK this can not be done for the other expandable row type due to the need for indenting.